samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties

```diff
@@ -42,6 +42,7 @@ cancel_question=Ezabatu galdera
 cancel_question_reduce_total=Murriztu puntuazio osoa
 cancel_question_distribute_points=Modu berean banatu puntuak
 cancel_question_cancel=Ez ezabatu galdera
+cancel_question_cancelled=Cancelled question
```

Comment:
@PABLOSANROMAN could you please contribute a translation? Thanks!

Reply:
Hi @mpellicer  

cancel_question_cancelled=Galdera baliogabetua

Reply:
Fixed, thanks a ton!